### ISSUE TITLE
Add support for Branding image (`windowSplashScreenBrandingImage`) 

### ISSUE THREAD

**Issue by JoniVR** [zoontek/react-native-bootsplash#284]
### Why it is needed?

Hi!

Thanks for your awesome work on this!

Currently, API 4.0.0 is making use of the new Splashscreen API on android. 
This API also allows you to set a branding image ([windowSplashScreenBrandingImage](https://developer.android.com/reference/android/R.attr#windowSplashScreenBrandingImage)).

Also mentioned: https://developer.android.com/guide/topics/ui/splash-screen 

I was wondering if you'd consider supporting this property in the future? 
Even though it's "not recommended" in Google's own guidelines, lots of modern enterprise apps do require some form of brand identity besides the app icon. 

Examples for this are all the ~~Facebook~~ Meta based apps, but also Microsoft apps on iOS.

I'd be awesome if this option would also place a branding image at a similar location for iOS, as this is also something that is done very often.	

### Possible implementation

From taking a first look into the codebase, this would probably have to be done in the `generate.ts` file that's used for the CLI if I'm understanding it enough? 

Basically adding another (optional) parameter that takes an image (and possibly a size parameter?), do almost the same thing as for the regular icon.

### Code sample

I don't have a code sample yet, but if you feel like you'd be okay with adding it, I might try to take some time to implement it and submit a PR. 

I just don't want to spend more effort into it if it's not something you'd accept anyways.

**Comment by zoontek** [comment]
This is not a priority for me and I don't really to support asset generation for that.
If someone want to have one, they will generate their assets and use `windowSplashScreenBrandingImage` directly.

**Comment by JoniVR** [comment]
Alright, thanks for the fast reply.

So if I were to submit a PR for it you wouldn't accept it? (if not that's okay, just wondering)

**Comment by zoontek** [comment]
No I wouldn't.
